core/functions/collection_funcs.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2025 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package functions
+
+import (
+	"context"
+	"sync"
+
+	"github.com/cockroachdb/errors"
+	"github.com/dolthub/dolt/go/store/types"
+
+	"github.com/dolthub/doltgresql/core/rootobject"
+	"github.com/dolthub/doltgresql/flatbuffers/gen/serial"
+)
+
+// CollectionFunctions contain the functions that are used for Collection to interact as a root object.
+type CollectionFunctions struct{}
+
+var _ rootobject.CollectionFunctions = CollectionFunctions{}
+
+// storage is used to read from and write to the root.
+var storage = rootobject.RootObjectSerializer{
+	Bytes:        (*serial.RootValue).FunctionsBytes,
+	RootValueAdd: serial.RootValueAddFunctions,
```

Comment:
You should comment in the flatbuffer table file what the value of this field is, `[ubyte]` could be anything.